cves/kernel/CVE-2022-3108.yml

```diff
@@ -106,7 +110,7 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 3a87177eb14113bbe8cd95a276af2c412eced6ac
-  note: Discovered automatically by archeogit.
+  note: Discovered automatically by archeogit. Manually Verified.
```

Comment:
I don't think you need "discovered automatically by archeogit." here anymore. You can just have "Manually Confirmed" if you have checked it.

Reply:
Fixing that